rockcraft/commands/init.py

```diff
@@ -129,7 +127,6 @@ class InitCommand(AppCommand):
                     most important story about it. Keep it under 100 words though,
                     we live in tweetspace and your description wants to look good in the
                     container registries out there.
-                license: GPL-3.0 # your application's SPDX license
```

Comment:
Is removing the license field from the init template necessary? I think it could be better to leave it there (maybe with a comment explaining that it's optional?) so one doesn't forget it if they want to make their rock public.

Reply:
@cjdcordeiro could/should we add the license as a commented-out line?

Reply:
that was a request from above